src/Core/Parser/SyntaxTree/Expression/MathExpressionNode.php

```diff
@@ -26,7 +26,7 @@ class MathExpressionNode extends AbstractExpressionNode
 		(
 			{                                # Start of shorthand syntax
 				(?:                          # Math expression is composed of...
-					[a-zA-Z0-9\.]+[\s]?[*+\^\/\%\-]{1}[\s]?[a-zA-Z0-9\.]+   # Various math expressions left and right sides with any spaces
+					[a-zA-Z0-9\.]+(?:[\s]?[*+\^\/\%\-]{1}[\s]?[a-zA-Z0-9\.]+)+   # Various math expressions left and right sides with any spaces
```

Comment:
In this expression are no returned matching groups. Multiple characters are good, but is should also match multiple groups. I tried, e.g. the following expressions:

{imageWidth / 16 * 9} worked, this matched as "imageWith / 1" and "6 * 9"
{imageWidth / 3 * 2} did not work.
{imageWidth / 3 + 7 * 5} did not work.

With the changed regular expression all of them work now.